titan/model.py

```diff
@@ -366,7 +368,7 @@ def exit(self):
         """
         Allow agents to exit model.
         """
-        if self.exits == {}:
+        if self.exits == {} or all(key == "none" for key in self.exits.keys()):
```

Comment:
since self.exits no filters out the `none` exit types, I think this or check is now redundant (and not strictly speaking correct as it's checking class name, not exit type)